tests/unit/conditions/test_return_value.py

```diff
@@ -23,6 +23,47 @@
 from nucypher.policy.conditions.lingo import ReturnValueTest
 
 
+def test_return_value_key():
+    test = ReturnValueTest(comparator=">", value="0", key="james")
+    assert test.eval({"james": 1})
+    assert not test.eval({"james": -1})
+
+    with pytest.raises(KeyError):
+        test.eval({"bond": 1})
+
+    test = ReturnValueTest(comparator=">", value="0", key=4)
+    assert test.eval({4: 1})
+    assert not test.eval({4: -1})
+
+    with pytest.raises(KeyError):
+        test.eval({5: 1})
+
+
+def test_return_value_index():
+    test = ReturnValueTest(comparator=">", value="0", key=0)
+    assert test.eval([1])
+    assert not test.eval([-1])
```

Comment:
Interesting situation here - in python a `TypeError` is raised when using operators (like `>` and `<`) of different types but it's not obvious where/how we resolve such cases with `ReturnValueTest` 🤔

Reply:
Not sure i understand the problem?

Reply:
I'm not sure if there's a problem, but I'm curios what the expected behavior is when comparing mismatched types (`1 > "2" for example.`.  In Python this is a `TypeError`.  Is the same true via `ast.literal_eval`?